prow/pod-utils/decorate/podspec.go

```diff
@@ -437,14 +450,14 @@ func processLog(log coreapi.VolumeMount, prefix string) string {
 	if prefix == "" {
 		return filepath.Join(log.MountPath, "process-log.txt")
 	}
-	return filepath.Join(log.MountPath, fmt.Sprintf("%s-log.txt", prefix))
+	return filepath.Join(log.MountPath, fmt.Sprintf("%s-build-log.txt", prefix))
 }
 
 func markerFile(log coreapi.VolumeMount, prefix string) string {
 	if prefix == "" {
 		return filepath.Join(log.MountPath, "marker-file.txt")
 	}
-	return filepath.Join(log.MountPath, fmt.Sprintf("%s-marker.txt", prefix))
+	return filepath.Join(log.MountPath, fmt.Sprintf("%s-marker-file.txt", prefix))
```

Comment:
nit: The changes to this function and `processLog` shouldn't be required and I don't think they improve clarity.